cron/sessionStoreExpire.sql

```diff
@@ -0,0 +1,4 @@
+-- BLOCK expire
+DELETE FROM pl_sessions
+WHERE EXTRACT(EPOCH FROM (now() - updated_at)) >= $expirationInSeconds
+;
```

Comment:
Nit: we conventionally don't put SQL semicolons on a separate line.

Reply:
On a multiline query, I prefer it alone/left-justified. It's easier to skim and know that it's there. If it's on the end of a long last line, like in the query above, it's not always as easy to verify that it's missing (until you get an SQL error).

Reply:
I can understand that, but I believe consistency is more important than personal preference. Eventually we'll have tooling enforcing consistent SQL formatting for us; for now, I unfortunately have to do it manually.

Reply:
#7151, so you don't need to do it manually...